homework-submission/todo-app/src/todos/element.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+
+class Element extends Component {
+    state = {
+        id: this.props.id,
+        description: this.props.description,
```

Comment:
Local state not necessary

Reply:
description is necessary , maybe id not okey!